constants/data.js

```diff
@@ -52,9 +52,10 @@ const userData = {
     title:
       "Software engineer with two years of full-time experience. I’ve worked across the stack, but especially enjoy building front-end experiences that help and delight people.",
     description: [
-      `Some more info about me ;)`,
-      `This is the perfect place to include some more info about me!`,
-      `I know how to code!`,
+      `👋🏼 Hi there! I'm Turner. Welcome to my site 🙂`,
+      `I'm a professional software engineer with two years' full-stack experience building web and mobile apps. I'm especially inspired by the positive impact possible at the intersection of tech, government, and civic engagement, but can find joy in any cause using software to help improve lives.`,
+      `When I'm not coding, you can find me at the gym, curled up with a good (or sometimes bad) sci-fi novel, or crafting new adventures for my D&D campaigns ⚔️🤓`,
+      `🙋‍♂️ I'm looking for a job! I thrive in environments with humble, curious, kind folks. If you think I might be a fit, please REACH OUT - I'd love to hear from you.`,
```

Comment:
Should `REACH OUT` be a mailto link? eg
```suggestion
      `🙋‍♂️ I'm looking for a job! I thrive in environments with humble, curious, kind folks. If you think I might be a fit, please <a href="mailto:jturnerallen@gmail.com"><strong>reach out</strong></a> - I'd love to hear from you.`,
```
(make sure ^that works 😅 )

Reply:
It doesn't work unfortunately. The bold and anchor tags just get displayed on the page as '<a>' etc.

The descriptions are mapped to the middle of <p> tags so I wanted to talk about the best way to go about fixing this.

I have updated the initial PR to include a picture of the working changes.

Reply:
Ok, look through this more thoroughly - since these are just dumped as text into <p> tags as strings [here](https://github.com/jtallen/turner-site/blob/about/components/AboutMe.js#L98-L105), you need add the last one directly into the `AboutMe` page. Something like:

```
            {userData.about.description?.map((desc, idx) => (
              <p
                key={idx}
                className="text-xl text-gray-700 mb-4 dark:text-gray-300 "
              >
                {desc}
              </p>
            ))}
            <p
                key={idx}
                className="text-xl text-gray-700 mb-4 dark:text-gray-300 "
              >
                🙋‍♂️ I'm looking for a job! I thrive in environments with humble, curious, kind folks. If you think I might be a fit, please <a href="mailto:jturnerallen@gmail.com"><strong>reach out</strong></a> - I'd love to hear from you.
              </p>
```